ibc-clients/ics07-tendermint/src/verifier.rs

```diff
@@ -0,0 +1,37 @@
+use tendermint_light_client_verifier::ProdVerifier;
+
+/// Specifies the Verifier interface that hosts must adhere to when customizing
+/// Tendermint client verification behaviour.
+///
+/// For users who require custom verification logic, i.e., in situations when
+/// the Tendermint `ProdVerifier` doesn't provide the desired outcome, users
```

Comment:
```suggestion
/// the Tendermint [`ProdVerifier`] doesn't provide the desired outcome, users
```

Can we also add links to other objects in the doc-strings? (It's enough to get the easy ones.)

Reply:
[274e0b1](https://github.com/cosmos/ibc-rs/pull/1166/commits/274e0b1c2e67f8a7f68ff23f1b3ce04cc5b16961)